src/components/Layout.tsx

```diff
@@ -24,7 +24,11 @@ export default function Layout({ children }: LayoutProps) {
             <div className="bodywrapper">
                 <div className="body">
                     <main className="main">{children}</main>
-                    <footer className="footer">About me</footer>
+                    <footer className="footer">
+                        <Link href="https://portfolio-site-k8ompbdj7-marvin-dems-projects.vercel.app/">
```

Comment:
That url refers to a specific build of your portfolio. You probably want it to always refer to the latest deployment though, so you should use the general domain of that project instead.